android/expoview/src/main/java/versioned/host/exp/exponent/modules/universal/notifications/ScopedServerRegistrationModule.java

```diff
@@ -8,14 +8,16 @@
 
 import expo.modules.notifications.serverregistration.ServerRegistrationModule;
 import host.exp.exponent.di.NativeModuleDepsProvider;
+import host.exp.exponent.kernel.ExperienceId;
 import host.exp.exponent.storage.ExponentSharedPreferences;
 
 public class ScopedServerRegistrationModule extends ServerRegistrationModule {
   @Inject
   ExponentSharedPreferences mExponentSharedPreferences;
 
-  public ScopedServerRegistrationModule(Context context) {
+  public ScopedServerRegistrationModule(Context context, ExperienceId experienceId) {
     super(context);
+    mInstallationId = new ScopedInstallationId(context);
```

Comment:
I believe this should be
```suggestion
    mInstallationId = new ScopedInstallationId(context, experienceId);
```

Reply:
Of course! So sorry, this was the last change which was not supposed to break anything, etc.

I have removed it as we do not need it (see https://github.com/expo/expo/pull/11249/commits/5ae0e9315e4204966345a3104430aa5f64e3a8bb)! `ScopedContext` does it for us if we're *not* in a standalone app! 🎉